meson.build

```diff
@@ -2719,7 +2719,9 @@ install_data('LICENSE.GPL2',
 install_subdir('LICENSES',
                install_dir : docdir)
 
+install_emptydir(systemenvgeneratordir)
 install_emptydir(systemdstatedir)
+install_emptydir(userenvgeneratordir)
```

Comment:
Please do this in core/meson.build, close to systemgeneratordir and friends

Reply:
Okay, I moved the changes to `src/core/meson.build`.